flocker/node/agents/blockdevice.py

```diff
@@ -1078,6 +1145,22 @@ def get_blockdevice_volume(api, blockdevice_id):
     raise UnknownVolume(blockdevice_id)
 
 
+def get_system_mounts():
+    """
+    Load information about mounted filesystems.  volumes.
```

Comment:
I'm guessing you wanted this to be "filesystems and volumes" rather than "filesystems.  volumes." but I could be wrong.